slices/liblua5.4-0.yaml

```diff
@@ -0,0 +1,18 @@
+package: liblua5.4-0
+
+essential:
+  - liblua5.4-0_copyright
+
+slices:
+  libs:
+    essential:
+      - libc6_libs
+      - libgcc-s1_libs
```

Comment:
I don't think `libgcc-s1` is declared as a dependency and thus should be removed:
```shell
$ apt show liblua5.4-0
...
Depends: libc6 (>= 2.38), libstdc++6 (>= 4.1.1)
...
```